src/components/Input/Input.d.ts

```diff
@@ -62,12 +62,6 @@ export interface InputProps {
 		 */
     type?: string;
 
-    /**
-		 * If `true`,  the input element is required
-		 * @default false
-		 */
-    required?: boolean;
-
```

Comment:
why we have removed this?

Reply:
Removed because we didn't have a design for the required state. 
Added again, will change the styling in the Enhancement phase if required.